data.sql

```diff
@@ -12,3 +12,9 @@ UPDATE animals SET owner_id = 2 WHERE name IN ('Gabumon', 'Pikachu');
 UPDATE animals SET owner_id = 3 WHERE name IN ('Devimon', 'Plantmon');
 UPDATE animals SET owner_id = 4 WHERE name IN ('Charmander', 'Squirtle', 'Blossom');
 UPDATE animals SET owner_id = 5 WHERE name IN ('Angemon', 'Boarmon');
+
+INSERT INTO vets (name, age, date_of_graduation) VALUES ('William Tatcher', 45, '2000-04-23'), ('Maisy Smith', 26, '2019-01-17'), ('Stephanie Mendez', 64, '1981-05-04'), ('Jack Harkness', 38, '2008-06-08');
+
+INSERT INTO specializations (vet_id, species_id) VALUES (1, 1), (3, 1), (3, 2), (4, 2);
```

Comment:
Here is a possible suggestion to say that to the student:

(Optional) I noticed that you used hardcoded values for vet_id and species_id in your queries. While this is not wrong, it is not a good practice, as it assumes that the values are fixed and known in advance. This may not be the case in a real-world scenario, where the data may change or be unknown. A better way of writing the queries is to use subqueries to select the id values based on the name columns. This would make your code more robust and accurate. For example, instead of writing:

INSERT INTO specializations (vet_id, species_id) VALUES (1, 1);

You could write:

INSERT INTO specializations (vet_id, species_id) VALUES ((SELECT id FROM vets WHERE name = 'William Tatcher'), (SELECT id FROM species WHERE name = 'Pokemon'));

I suggest that you try to use subqueries instead of hardcoded values for vet_id and species_id in your queries. This will help you improve your SQL skills and prepare you for more complex scenarios.